src/LICENCE.md

```diff
@@ -0,0 +1,7 @@
+Copyright <2018> <Mera Gangapersaud>
```

Comment:
Check out how this renders:

https://github.com/Mera-Gangapersaud/Creative-Collab/blob/651c55f23e0e3cbde7c334ff34afffd486e8b16b/src/LICENCE.md

You'll notice that Mera's name is missing! 🙀

We'll need to update this to `Copyright 2018 Mera Gangapersaud`

Reply:
I will work on that 👍🏻😊